Userland/Applets/ClipboardHistory/ClipboardHistoryModel.h

```diff
@@ -41,6 +44,11 @@ class ClipboardHistoryModel final : public GUI::Model
     void clear();
     bool is_empty() { return m_history_items.is_empty(); }
 
+    ErrorOr<void> read_from_file(DeprecatedString const& path);
```

Comment:
not sure if model should take care of files… maybe this would be better extracted somewhere instead? similarly json serde? (those could probably be even placed in one location — you can clearly see itʼs a nice self-contained feature [the clipboard is perfectly running without])

Reply:
The issue is that the `clipboard_content_did_change` function is inside the model. So implementing this feature would either require to modify the model or making an external `ModelClient` which seems overkill IMO.
And I don't see why it would be better to have free functions instead of methods for serialization.

Reply:
well… thatʼs first time we serialise clipboard content; maybe what iʼm suggesting can wait (so the current implementation would be a canonical one). iʼm not strictly about free functions; more a serde feature that uses clipboard—this is only a question whether clipboard history shall be concerned with serde or is it something external to it. i just like that clipboard history model is all about clipboard history.

if such thing will be needed, itʼll definitely be implemented or specifically requested then. consider this issue as resolved. 😁